src/components/timeline/form/TimelineDetailsPanel.svelte

```diff
@@ -1,48 +1,48 @@
 <svelte:options immutable={true} />
 
 <script lang="ts">
+  import ArrowLeftIcon from '@nasa-jpl/stellar/icons/arrow_left.svg?component';
   import PenIcon from '@nasa-jpl/stellar/icons/pen.svg?component';
   import PlusIcon from '@nasa-jpl/stellar/icons/plus.svg?component';
   import TrashIcon from '@nasa-jpl/stellar/icons/trash.svg?component';
   import GripVerticalIcon from 'bootstrap-icons/icons/grip-vertical.svg?component';
   import { onMount } from 'svelte';
   import { dndzone } from 'svelte-dnd-action';
-  import { get } from 'svelte/store';
-  import DatePickerField from '../../../components/form/DatePickerField.svelte';
   import Input from '../../../components/form/Input.svelte';
-  import { field } from '../../../stores/form';
+  import DatePicker from '../../../components/ui/DatePicker/DatePicker.svelte';
   import { maxTimeRange, viewTimeRange } from '../../../stores/plan';
   import {
+    selectedRow,
+    selectedRowId,
     selectedTimeline,
     selectedTimelineId,
     view,
+    viewSetSelectedRow,
     viewSetSelectedTimeline,
+    viewUpdateRow,
     viewUpdateTimeline,
   } from '../../../stores/views';
   import { getTarget } from '../../../utilities/generic';
   import { getDoyTime } from '../../../utilities/time';
-  import { createRow, createVerticalGuide } from '../../../utilities/timeline';
+  import { createHorizontalGuide, createRow, createVerticalGuide } from '../../../utilities/timeline';
   import { tooltip } from '../../../utilities/tooltip';
-  import { required, timestamp } from '../../../utilities/validators';
   import GridMenu from '../../menus/GridMenu.svelte';
   import CssGrid from '../../ui/CssGrid.svelte';
   import Panel from '../../ui/Panel.svelte';
 
   export let gridId: number;
 
   let rows: Row[] = [];
-  let verticalGuideFieldMap: Record<number, FieldStore<string>> = {};
 
   $: rows = $selectedTimeline?.rows || [];
   $: verticalGuides = $selectedTimeline?.verticalGuides || [];
+  $: horizontalGuides = $selectedRow?.horizontalGuides || [];
+  $: rowHasNonActivityChartLayer = !!$selectedRow?.layers.find(layer => layer.chartType !== 'activity') || false;
 
-  $: if (verticalGuides) {
-    verticalGuideFieldMap = verticalGuides.reduce((map, guide) => {
-      if (!map[guide.id]) {
-        map[guide.id] = field<string>(guide.timestamp, [required, timestamp]);
-      }
-      return map;
-    }, {});
+  function updateRowEvent(event: Event) {
+    event.stopPropagation();
```

Comment:
Not 100% necessary but instead of calling `event.stopPropagation()` here you can do it on the markup below:

```svelte
<input on:input|stopPropagation={updateRowEvent} />
```